README.md

```diff
@@ -125,6 +125,55 @@ gopay_charge = midtrans.gopay_charge.post(charge_params)
 #=> gopay_charge returns MidtransApiMidtransApi::Model::Gopay::Charge instance
 ```
 
+#### BCA Virtual Account Charge
+```ruby
+charge_params = {
+  "payment_type": "bank_transfer",
+  "transaction_details": {
+      "order_id": "order-with-bca-virtua-account",
+      "gross_amount": 12500
+  },
+  "item_details": [
```

Comment:
just curious, why do we need to define the item details when creating the VA number?

Reply:
base on midtrans API documentation ([https://api-docs.midtrans.com/?php#bca-virtual-account](https://api-docs.midtrans.com/?php#bca-virtual-account)) that's marked as a mandatory field with price, quantity, and name as a mandatory field for each item

Reply:
roger that